<!doctype html>
<!-- 
This publication template is based on the one created by Andrew G York: https://github.com/AndrewGYork/publication_template . Thanks Andrew!

Based on this theme: https://github.com/broccolini/dinky , which mentioned that attribution is appreciated. Thanks, broccolini!
-->
<html lang="en">
  <head>
    <base target="_blank">
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="citation_title" content="mScarlet fluorescence lifetime reports lysosomal pH with high precision">
    <meta name="citation_author" content="Julia R. Lazzari-Dean">
    <meta name="citation_author" content="Maria Clara Ingaramo">
    <meta name="citation_author" content="John C.K. Wang">
    <meta name="citation_author" content="John Yong">
    <meta name="citation_author" content="Maria Ingaramo">
    <meta name="citation_publication_date" content="2022/03/24">
    <meta name="citation_journal_title" content="Github.io">
    <meta name="citation_pdf_url" content="https://jlazzaridean.github.io/mScarlet_lifetime_reports_pH/mScarlet_lysosomal_pH.pdf">
    <title>mScarlet fluorescence lifetime reports lysosomal pH with high precision</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/prism.css">

    <script src="javascript/scale-fix/scale.fix.js"></script>
    <script src="javascript/python-highlighting/prism.js"></script>
    <script async  src="javascript/Minimal-MathJax/MathJax.js?config=TeX-AMS_CHTML"></script>
    <script src="javascript/update_figures.js"></script>
    <script src="javascript/reference_list/reference_list.js"></script>
    <script src="javascript/google-analytics/analytics.js"></script><!--run AGY analytics tag-->
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
  </head>
  <body>
    <div class="wrapper">
      <header>
        <h1 class="header">mScarlet fluorescence lifetime</h1>
        <ul>
          <li class="download"><a class="buttons" href="https://github.com/jlazzaridean/mScarlet_lifetime_reports_pH/zipball/main">Download ZIP</a></li>
          <li class="download"><a class="buttons github" href="https://github.com/jlazzaridean/mScarlet_lifetime_reports_pH">View On GitHub</a></li>
          <li class="download"><a class="buttons pdf" href="./mScarlet_lysosomal_pH.pdf">Download PDF</a></li>
        </ul>
        <p class="header">This project is maintained by <a class="header name" href="https://jlazzaridean.github.io">Julia Lazzari-Dean</a> in the <a class="header name" href="http://andrewgyork.github.io/">York lab</a>, and was funded by <a class="header name" href="https://www.calicolabs.com/">Calico Life Sciences LLC</a></p>
      </header>
<!-- 
##############################################################
The real text begins here.
##############################################################
 -->
<section>
<h1>Research article</h1>
<h2>mScarlet fluorescence lifetime reports lysosomal pH with high precision</h2>
<p class="author_list">Julia R. Lazzari-Dean (<a href="https://orcid.org/0000-0003-2971-5379">ORCID</a>),<sup>*</sup> Maria Clara Ingaramo, John C.K. Wang, John Yong, Maria Ingaramo<sup>&dagger;</sup></p>
<p class="author_affiliations">Calico Life Sciences LLC, South San Francisco, CA 94080, USA</p>
<p class="contact_email"><sup>*</sup>Permanent email: <a href="mailto:julia.lazzaridean+mScarlet_lifetime@gmail.com" target="_self">julia.lazzaridean+mScarlet_lifetime@gmail.com</a></p>
<p class="contact_email"><sup>&dagger;</sup>Permanent email: <a href="mailto:maria.del.mar.ingaramo+mScarlet_lifetime@gmail.com" target="_self">maria.del.mar.ingaramo+mScarlet_lifetime@gmail.com</a></p>

<p></p>
<p><strong>Pre-print published: March 24, 2022</strong></p>
<p><strong>You are currently reading a rendered HTML version of a <a href="https://doi.org/10.5281/zenodo.6363342">Zenodo publication</a>. When citing this work, please refer to the Zenodo version and cite as:</strong> Julia R. Lazzari-Dean, Maria Clara Ingaramo, John C.K. Wang, John Yong, Maria Ingaramo. mScarlet fluorescence lifetime reports lysosomal pH with high precision. Zenodo (2022) <a href="https://doi.org/10.5281/zenodo.6363342">DOI: 10.5281/zenodo.6363342</a></p>

<p></p>
<img src="./figures/TOC_graphic/TOC_graphic_2.png" alt="TOC graphics" id="table_of_contents_image">
<p></p>

<a id="Abstract" href="#Abstract" target="_self">
<h4>Abstract</h4></a>
<p class="abstract">The lysosome maintains a highly acidic pH, which is critical for successful lysosomal catabolism. Lysosomal pH (pHlys) is difficult to measure because of the simultaneous need for a sensor with large dynamic range, genetic targetability, low pKa, and a quantitative readout. Here, we demonstrate that the fluorescence lifetime of the mScarlet-LAMP1 fusion protein quantitatively reports lysosomal pH, exhibiting a large dynamic range and a pKa well-tuned for the lysosome. Because fluorescence lifetime is an intrinsic property, pH measurements can be achieved in a single fluorescence channel. mScarlet-LAMP1 lifetime allows for individual lysosome-resolved recordings, a critical advance in describing and understanding lysosomal pHlys heterogeneity. Using this biosensor, we quantify heterogeneity of pHlys in cultured cells at rest and over time during drug treatment. We anticipate that mScarlet-LAMP1 will enable new insights into the diversity of lysosomal physiology and ionic milieu.</p>

<h3 class="onlyprint">
Note that this is a limited PDF or print version; animated and interactive figures are disabled. For the full version of this article, please visit <a href="https://andrewgyork.github.io/mScarlet_lifetime_reports_pH">https://andrewgyork.github.io/mScarlet_lifetime_reports_pH</a></h3>

<noscript>
<h3>Your browser doesn't seem to support Javascript. This document uses Javascript for interactive figures, math typesetting, and to automatically generate the reference list. Either activate Javascript, or use the "Download PDF" link above if you want properly typeset math and a reference list.</h3></noscript>

<h3>Introduction</h3>

<p>Lysosomes are key organelles in nutrient homeostasis. They are the site of recycling and digestion of macromolecules, and they help to sense and maintain the balance between anabolic and catabolic processes

[<a class="citation" href="http://doi.org/10.1016/j.tibs.2018.10.006" title="Lysosomal Ion Channels as Decoders of Cellular Signals; Ping Li, Mingxue Gu, Haoxing Xu; Trends in Biochemical Sciences 44, 2, 110-124 (2018)">Li 2018</a>].

The lysosomal lumen is highly acidic (pH \(\approx\) 5) relative to the cytoplasm (pH \(\approx\) 7), which sets up a proton gradient across the lysosomal membrane

[<a class="citation" href="http://doi.org/10.1146/annurev-physiol-012110-142317" title="Lysosomal Acidification Mechanisms; Joseph A. Mindell; Annual Review of Physiology 74, 1, 69-86 (2012)">Mindell 2012</a>].

This proton gradient is largely maintained by the lysosomal V-ATPase, an ATP-driven proton pump, but recent work has also described a diversity of ionic currents, many without identified channels or transporters

[<a class="citation" href="http://doi.org/10.1146/annurev-physiol-012110-142317" title="Lysosomal Acidification Mechanisms; Joseph A. Mindell; Annual Review of Physiology 74, 1, 69-86 (2012)">Mindell 2012</a>, 

<a class="citation" href="http://doi.org/10.1146/annurev-physiol-021014-071649" title="Lysosomal Physiology; Haoxing Xu, Dejian Ren; Annual Review of Physiology 77, 1, 57-80 (2015)">Xu 2015</a>].

Low lysosomal pH (pHlys) is necessary for proper function, as most of the lysosomal hydrolases require an acidic environment

[<a class="citation" href="http://doi.org/10.1146/annurev-physiol-021014-071649" title="Lysosomal Physiology; Haoxing Xu, Dejian Ren; Annual Review of Physiology 77, 1, 57-80 (2015)">Xu 2015</a>].

Misregulation of lysosomal pH has been shown to cause inflammation and iron deficiency

[<a class="citation" href="http://doi.org/10.7554/elife.51031" title="Impaired Lysosomal Acidification Triggers Iron Deficiency and Inflammation in Vivo; King Faisal Yambire, Christine Rostosky, Takashi Watanabe, David Pacheu-Grau, Sylvia Torres-Odio, Angela Sanchez-Guerrero, Ola Senderovich, Esther G Meyron-Holtz, Ira Milosevic, Jens Frahm, A Phillip West, Nuno Raimundo; eLife 8, e51031 (2019)">Yambire 2019</a>],

and it has been linked to Parkinson’s disease and other progressive age-related disorders

[<a class="citation" href="http://doi.org/10.1016/j.tibs.2018.10.006" title="Lysosomal Ion Channels as Decoders of Cellular Signals; Ping Li, Mingxue Gu, Haoxing Xu; Trends in Biochemical Sciences 44, 2, 110-124 (2018)">Li 2018</a>, 

<a class="citation" href="http://doi.org/10.1016/j.bbapap.2020.140443" title="The aging lysosome: An essential catalyst for late-onset neurodegenerative diseases; Ralph A. Nixon; BBA - Proteins and Proteomics 1868, 9, 140443 (2020)">Nixon 2020</a>, 

<a class="citation" href="http://doi.org/10.1016/j.celrep.2015.07.050" title="Presenilin 1 Maintains Lysosomal Ca2+ Homeostasis via TRPML1 by Regulating vATPase-Mediated Lysosome Acidification; Ju-Hyun Lee, Mary Kate McBrayer, Devin M. Wolfe, Luke J. Haslett, Asok Kumar, Yutaka Sato, Pearl P.Y. Lie, Panaiyur Mohan, Erin E. Coffey, Uday Kompella, Claire H. Mitchell, Emyr Lloyd-Evans, Ralph A. Nixon; Cell Reports 12, 9, 1430-1444 (2015)">Lee 2015</a>].

</p>

<p>Further adding to the complexity of lysosome biology, a typical mammalian cell contains hundreds of lysosomes, each with potentially diverse properties and roles. Heterogeneity among lysosomes has been described for many attributes, including size, location within the cell

[<a class="citation" href="http://doi.org/10.1111/tra.12587" title="Mechanisms of Lysosomal Positioning and Movement; Birol Cabukusta, Jacques Neefjes; Traffic 19, 761-769 (2018)">Cabukusta 2018</a>],

and movement speed

[<a class="citation" href="http://doi.org/10.1021/acs.analchem.8b01563" title="Characterize Collective Lysosome Heterogeneous Dynamics in Live Cell with a Space- and Time-Resolved Method; Hansen Zhao, Qiming Zhou, Mengchan Xia, Jiaxin Feng, Yang Chen, Sichun Zhang, Xinrong Zhang; Analytical Chemistry 90, 15 (2018)">Zhao 2018</a>].

For pH specifically, progressive acidification along the endolysosomal pathway has been reported

[<a class="citation" href="http://doi.org/10.1146/annurev.cellbio.12.1.575" title="Endocytosis and Molecular Sorting; Ira Mellman; Annual Review of Cell and Developmental Biology 12, 575-625 (1996)">Mellman 1996</a>],

as well as some heterogeneity in pH among lysosomes themselves

[<a class="citation" href="http://doi.org/10.1016/j.cub.2016.06.046" title="Endolysosomes Are the Principal Intracellular Sites of Acid Hydrolase Activity; Nicholas A. Bright, Luther J. Davis, J. Paul Luzio; Current Biology 26, 17, 2233-2245 (2016)">Bright 2016</a>].

Some studies have also reported spatial patterning of pHlys within cells

[<a class="citation" href="http://doi.org/10.1083/jcb.201507112" title="The Position of Lysosomes within the Cell Determines Their Luminal pH; Danielle E. Johnson, Philip Ostrowski, Valentin Jaumouillé, Sergio Grinstein; Journal of Cell Biology 212, 6, 667-692 (2016)">Johnson 2016</a>, 

<a class="citation" href="http://doi.org/10.1091/mbc.e20-06-0383" title="pHLARE: A New Biosensor Reveals Decreased Lysosome pH in Cancer Cells; Bradley A. Webb, Francesca M. Aloisio, Rabab A. Charafeddine, Jessica Cook, Torsten Wittmann, Diane L. Barber; Molecular Biology of the Cell 32, 2, 131-142 (2021)">Webb 2021</a>],

although others have not seen such distributions

[<a class="citation" href="http://doi.org/10.1080/15548627.2020.1771858" title="Live Imaging of Intra-Lysosome pH in Cell Lines and Primary Neuronal Culture Using a Novel Genetically Encoded Biosensor; Amy H. Ponsford, Thomas A. Ryan, Andrea Raimondi, Emanuele Cocucci, Susanne A. Wycislo, Florian Fröhlich, Laura E. Swan, Massimiliano Stagi; Autophagy 17, 6, 1500-1518 (2020)">Ponsford 2020</a>].

Such discrepancies, and our incomplete description of pHlys distributions in space and time, stem in part from limitations in quantification possible with current pHlys sensors.</p>

<p>Despite its biological importance, measuring pHlys at the individual lysosome level in live cells remains challenging
 
[<a class="citation" href="http://doi.org/10.1111/ejn.12169" title="Autophagy Failure in Alzheimer’s Disease and the Role of Defective Lysosomal Acidification; Devin M. Wolfe, Ju-hyun Lee, Asok Kumar, Sooyeon Lee, Samantha J. Orenstein, Ralph A. Nixon; European Journal of Neuroscience 37, 12, 1949-1961 (2013)">Wolfe 2013</a>].

To our knowledge, no existing pHlys sensor excels in all four of the following categories, which fundamentally limits the scope of pHlys measurements.</p>

<ol>
<li>The sensor must be selectively and stably targeted to the lysosome over other acidic compartments. Many existing probes are dextran or DNA-conjugated dyes

[<a class="citation" href="http://doi.org/10.1038/s41565-018-0318-5" title="A DNA Nanomachine Chemically Resolves Lysosomes in Live Cells; KaHo Leung, Kasturi Chakraborty, Anand Saminathan, Yamuna Krishnan; Nature Nanotechnology 14, 2, 176-183 (2019)">Leung 2019</a>],

which are endocytosed and reach the lysosome by traversing the endocytic pathway. This process is not inherently specific for the lysosome and requires long (approximately 24 hour) incubation periods before the signal can be read out

[<a class="citation" href="http://doi.org/10.1111/ejn.12169" title="Autophagy Failure in Alzheimer’s Disease and the Role of Defective Lysosomal Acidification; Devin M. Wolfe, Ju-hyun Lee, Asok Kumar, Sooyeon Lee, Samantha J. Orenstein, Ralph A. Nixon; European Journal of Neuroscience 37, 12, 1949-1961 (2013)">Wolfe 2013</a>].

In comparison, genetically encoded tools have considerable advantages in targeting specificity and stability, but they often suffer in other areas.</li> 

<li>The sensor should have a pKa close to the pH value of the lysosome (pH \(\approx\) 5), but most existing pH sensors were developed for the cytosol (pH \(\approx\) 7).</li>

<li>The fast movement of lysosomes

[<a class="citation" href="http://doi.org/10.1146/annurev-physiol-021014-071649" title="Lysosomal Physiology; Haoxing Xu, Dejian Ren; Annual Review of Physiology 77, 1, 57-80 (2015)">Xu 2015</a>]

requires that the measurements be made relatively quickly. This need for speed, combined with the lysosomes’ small size (100 nm - 1 &mu;m), mandates a bright sensor with a large dynamic range, preferably in a single spectral channel.</li>

<li>The sensor should be able to report absolute pH, not just track pH changes. This requires a “self-normalizing” readout that is independent of the sensor concentration (explained further below).</li>
</ol>

<p>Fluorescence intensity of a single-color sensor is sometimes used to record pH, but it is not “self-normalizing.” Many biosensors change their fluorescence intensity (brightness) in response to pH, and intensity can be conveniently measured on many microscopes. However, fluorescence intensity is an extrinsic property, meaning that the signal from 10 fluorophores with 1x brightness can be confused with the signal from 5 fluorophores with 2x brightness. Furthermore, properties of the instrument also affect fluorescence intensity, e.g. the amount of illumination light applied to the sample and the efficiency with which the resulting fluorescence is detected. Therefore, without a self-normalizing strategy, intensity alone is a poor proxy for pH.</p>

<p>One strategy to improve quantitation is to add a second, spectrally distinct fluorophore to correct for sensor concentration 

[<a class="citation" href="http://doi.org/10.1091/mbc.e20-06-0383" title="pHLARE: A New Biosensor Reveals Decreased Lysosome pH in Cancer Cells; Bradley A. Webb, Francesca M. Aloisio, Rabab A. Charafeddine, Jessica Cook, Torsten Wittmann, Diane L. Barber; Molecular Biology of the Cell 32, 2, 131-142 (2021)">Webb 2021</a>, 

<a class="citation" href="http://doi.org/10.1080/15548627.2020.1771858" title="Live Imaging of Intra-Lysosome pH in Cell Lines and Primary Neuronal Culture Using a Novel Genetically Encoded Biosensor; Amy H. Ponsford, Thomas A. Ryan, Andrea Raimondi, Emanuele Cocucci, Susanne A. Wycislo, Florian Fröhlich, Laura E. Swan, Massimiliano Stagi; Autophagy 17, 6, 1500-1518 (2020)">Ponsford 2020</a>,

<a class="citation" href="http://doi.org/10.1021/acssensors.0c02318" title="Genetically Encoded, pH-Sensitive mTFP1 Biosensor for Probing Lysosomal pH; Marcus Y. Chin, Anand R. Patwardhan, Kean-Hooi Ang, Austin L. Wang, Carolina Alquezar, Mackenzie Welch, Phi T. Nguyen, Michael Grabe, Anna V. Molofsky, Michelle R. Arkin, Aimee W. Kao; ACS Sensors 6, 6, 2168-2180 (2021)">Chin 2021</a>].

For this to be accurate, the second fluorophore must be (1) pH independent, (2) at known stoichiometry with the sensing fluorophore, and (3) stable over the course of the measurement. These two-color tools can be deployed relatively easily on simple fluorescence microscopes, making them attractive for higher throughput measurements. However, the above assumptions are often violated, leading to potentially erroneous results. Most (if not all) fluorescent proteins exhibit some pH dependence, and the lysosome is a proteolytic compartment, making the likelihood of cleavage of at least one of the two fluorophores fairly high. In addition, differential photobleaching generally leads to loss of one fluorophore faster than the other over time

[<a class="citation" href="http://doi.org/10.1016/j.bpj.2013.11.4493" title="Temporal Dynamics of Microbial Rhodopsin Fluorescence Reports Absolute Membrane Voltage; Jennifer H. Hou, Veena Venkatachalam, Adam E. Cohen; Biophysical Journal 106, 3, 639-648 (2014)">Hou 2014</a>].

</p>

<p>A more reliable self-normalizing approach is to record the fluorescence lifetime instead of the fluorescence intensity of the sensor. Fluorescence lifetime, or the amount of time that a fluorophore remains in the excited state before emitting a photon and decaying to the ground state, is largely independent of fluorophore concentration and illumination intensity

[<a class="citation" href="http://doi.org/10.1016/j.cbpa.2015.05.024" title="Quantitative Two-Photon Imaging of Fluorescent Biosensors; Gary Yellen, Rebecca Mongeon; Current Opinion in Chemical Biology 27, 24-30 (2015)">Yellen 2015</a>].

Lifetime is related to quantum yield and is increasingly being deployed in a sensing context (e.g. 

[<a class="citation" href="http://doi.org/10.1016/j.cbpa.2015.05.024" title="Quantitative Two-Photon Imaging of Fluorescent Biosensors; Gary Yellen, Rebecca Mongeon; Current Opinion in Chemical Biology 27, 24-30 (2015)">Yellen 2015</a>, 

<a class="citation" href="http://doi.org/10.7554/elife.44522" title="Optical Estimation of Absolute Membrane Potential Using Fluorescence Lifetime Imaging; Julia R. Lazzari-Dean, Anneliese M.M. Gest, Evan W. Miller; eLife 8, e44522 (2019)">Lazzari-Dean 2019</a>,

<a class="citation" href="http://doi.org/10.1038/nature07842" title="Activation of CaMKII in Single Dendritic Spines during Long-Term Potentiation; Seok-Jin R. Lee, Yasmin Escobedo-Lozoya, Erzsebet M. Szatmari, Ryohei Yasuda; Nature 458, 7236, 299-304 (2009)">Lee 2009</a>,

<a class="citation" href="http://doi.org/10.1016/j.neuron.2015.09.043" title="Time-Resolved Imaging Reveals Heterogeneous Landscapes of Nanomolar Ca2+ in Neurons and Astroglia; Kaiyu Zheng, Lucie Bard, James P. Reynolds, Claire King, Thomas P. Jensen, Alexander V. Gourine, Dmitri A. Rusakov; Neuron 88, 2, 277-288 (2015)">Zheng 2015</a>]);  

it can be measured on specialized microscopes and flow cytometers. There has been considerable interest in the use of fluorescent proteins’ lifetime to measure cytosolic pH

[<a class="citation" href="http://doi.org/10.1016/j.bbabio.2014.04.003" title="EGFP-pHsens as a Highly Sensitive Fluorophore for Cellular pH Determination by Fluorescence Lifetime Imaging Microscopy (FLIM); Franz-Josef Schmitt, Bastian Thaa, Cornelia Junghans, Marco Vitali, Michael Veit, Thomas Friedrich; Biochimica et Biophysica Acta 1837, 1581-1593 (2014)">Schmitt 2014</a>, 

<a class="citation" href="http://doi.org/10.1007/s00216-013-6860-y" title="The Enhanced Cyan Fluorescent Protein: A Sensitive pH Sensor for Fluorescence Lifetime Imaging; Sandrine Poëa-Guyon, Hélène Pasquier, Fabienne Mérola, Nicolas Morel, Marie Erard; Analytical and Bioanalytical Chemistry 405, 12, 3983-3987 (2013)">Poëa-Guyon 2013</a>, 

<a class="citation" href="http://doi.org/10.1021/acs.analchem.9b01562" title="Quantifying Acute Fuel and Respiration Dependent pH Homeostasis in Live Cells Using the mCherryTYG Mutant as a Fluorescence Lifetime Sensor; Emily P. Haynes, Megha Rajendran, Chace K. Henning, Abhipri Mishra, Angeline M. Lyon, Mathew Tantama; Analytical Chemistry 91, 13, 8466-8475 (2019)">Haynes 2019</a>, 

<a class="citation" href="http://doi.org/10.1021/acschembio.1c00907" title="Fluorescence Lifetime Imaging of pH along the Secretory Pathway; Peter T. A. Linders, Melina Ioannidis, Martin ter Beest, Geert van den Bogaart; ACS Chemical Biology 17, 1, 240-251 (2022)">Linders 2022</a>],

but these implementations all have pKas close to neutral pH (>6), and many suffer from limited dynamic range (&lt;0.5 ns total change in lifetime). </p>

<p>Here, we investigate fluorescence lifetime imaging (FLIM) of the protein <a href="https://www.fpbase.org/protein/mscarlet/">mScarlet</a>

[<a class="citation" href="http://doi.org/10.1038/nmeth.4074" title="mScarlet: A Bright Monomeric Red Fluorescent Protein for Cellular Imaging; Daphne S Bindels, Lindsay Haarbosch, Laura van Weeren, Marten Postma, Katrin E Wiese, Marieke Mastop, Sylvain Aumonier, Guillaume Gotthard, Antoine Royant, Mark A Hink, Theodorus W J Gadella Jr; Nature Methods 14, 1, 53-56 (2017)">Bindels 2017</a>]  

for pH sensing in lysosomes. We note that the use of mScarlet or its derivatives to sense pH has been explored in other contexts: pHmScarlet, a derivative of mScarlet with a pKa of 7, has been reported

[<a class="citation" href="http://doi.org/10.1038/s41467-021-21666-7" title="pHmScarlet Is a pH-Sensitive Red Fluorescent Protein to Monitor Exocytosis Docking and Fusion Steps; Anyuan Liu, Xiaoshuai Huang, Wenting He, Fudong Xue, Yanrui Yang, Jiajia Liu, Liangyi Chen, Lin Yuan, Pingyong Xu; Nature Communications 12, 1, 1413 (2021)">Liu 2021</a>],

and some of the authors here recently investigated mScarlet lifetime in yeast

[<a class="citation" href="http://doi.org/10.1091/mbc.e21-06-0314" title="Loss of Major Nutrient Sensing and Signaling Pathways Suppresses Starvation Lethality in Electron Transport Chain Mutants; Alisha G. Lewis, Robert Caldwell, Jason V. Rogers, Maria Ingaramo, Rebecca Y. Wang, Ilya Soifer, David G. Hendrickson, R. Scott McIsaac, David Botstein, Patrick A. Gibney; Molecular Biology of the Cell 32, 22, ar39 (2021)">Lewis 2021</a>].

To our knowledge, however, mScarlet has not been previously characterized and deployed as a pHlys reporter.</p>

<p>In this work, we show that mScarlet has a large dynamic range and an ideally tuned pKa for pHlys recordings (reported pKa of 5.3

[<a class="citation" href="http://doi.org/10.1038/nmeth.4074" title="mScarlet: A Bright Monomeric Red Fluorescent Protein for Cellular Imaging; Daphne S Bindels, Lindsay Haarbosch, Laura van Weeren, Marten Postma, Katrin E Wiese, Marieke Mastop, Sylvain Aumonier, Guillaume Gotthard, Antoine Royant, Mark A Hink, Theodorus W J Gadella Jr; Nature Methods 14, 1, 53-56 (2017)">Bindels 2017</a>]). 

This, together with its genetic targetability and red-shifted spectrum, makes it a best-in-class pHlys sensor. We demonstrate the use of mScarlet lifetime in monitoring the effects of media change or V-ATPase inhibition on pHlys both in endpoint assays and longitudinally. Notably, mScarlet lifetime is able to resolve pHlys at the single lysosome level, uncovering dramatic pHlys heterogeneity within individual cells at steady state and in response to perturbation. We anticipate that mScarlet lifetime will open up new avenues in studies of pHlys and its heterogeneity.</p>

<h3>Results and Discussion</h3>

<h4>In vitro characterization of mScarlet lifetime</h4>

<p>We selected mScarlet as a low pH sensing domain because of its reported pKa of 5.3 and its long lifetime

[<a class="citation" href="http://doi.org/10.1038/nmeth.4074" title="mScarlet: A Bright Monomeric Red Fluorescent Protein for Cellular Imaging; Daphne S Bindels, Lindsay Haarbosch, Laura van Weeren, Marten Postma, Katrin E Wiese, Marieke Mastop, Sylvain Aumonier, Guillaume Gotthard, Antoine Royant, Mark A Hink, Theodorus W J Gadella Jr; Nature Methods 14, 1, 53-56 (2017)">Bindels 2017</a>],

as well as results from a preliminary screen we conducted in yeast (see <a href="./appendix.html#yeast_screen_text">Supplementary Text</a>). To further investigate mScarlet as a lifetime-based pHlys sensor, we characterized its pH sensitivity in vitro using fluorescence lifetime imaging (FLIM). We record a pKa of 5.6 for the lifetime of purified mScarlet (<a href="#pH_sensitivity_main_figure" target="_self">Figure 1A</a>), in good agreement with previous reports

[<a class="citation" href="http://doi.org/10.1038/nmeth.4074" title="mScarlet: A Bright Monomeric Red Fluorescent Protein for Cellular Imaging; Daphne S Bindels, Lindsay Haarbosch, Laura van Weeren, Marten Postma, Katrin E Wiese, Marieke Mastop, Sylvain Aumonier, Guillaume Gotthard, Antoine Royant, Mark A Hink, Theodorus W J Gadella Jr; Nature Methods 14, 1, 53-56 (2017)">Bindels 2017</a>].

We also observed a nearly two-fold increase in lifetime from pH 4 to pH 7.5 (mean arrival times of 2.05 ns and 3.65 ns respectively). Although various FPs have been previously demonstrated to have pH sensitive lifetimes

[<a class="citation" href="http://doi.org/10.1016/j.bbabio.2014.04.003" title="EGFP-pHsens as a Highly Sensitive Fluorophore for Cellular pH Determination by Fluorescence Lifetime Imaging Microscopy (FLIM); Franz-Josef Schmitt, Bastian Thaa, Cornelia Junghans, Marco Vitali, Michael Veit, Thomas Friedrich; Biochimica et Biophysica Acta 1837, 1581-1593 (2014)">Schmitt 2014</a>, 

<a class="citation" href="http://doi.org/10.1007/s00216-013-6860-y" title="The Enhanced Cyan Fluorescent Protein: A Sensitive pH Sensor for Fluorescence Lifetime Imaging; Sandrine Poëa-Guyon, Hélène Pasquier, Fabienne Mérola, Nicolas Morel, Marie Erard; Analytical and Bioanalytical Chemistry 405, 12, 3983-3987 (2013)">Poëa-Guyon 2013</a>, 

<a class="citation" href="http://doi.org/10.1021/acs.analchem.9b01562" title="Quantifying Acute Fuel and Respiration Dependent pH Homeostasis in Live Cells Using the mCherryTYG Mutant as a Fluorescence Lifetime Sensor; Emily P. Haynes, Megha Rajendran, Chace K. Henning, Abhipri Mishra, Angeline M. Lyon, Mathew Tantama; Analytical Chemistry 91, 13, 8466-8475 (2019)">Haynes 2019</a>, 

<a class="citation" href="http://doi.org/10.1021/acschembio.1c00907" title="Fluorescence Lifetime Imaging of pH along the Secretory Pathway; Peter T. A. Linders, Melina Ioannidis, Martin ter Beest, Geert van den Bogaart; ACS Chemical Biology 17, 1, 240-251 (2022)">Linders 2022</a>],

we note that this total change of 1.6 ns in mean arrival time from pH 4 to pH 7.5 is the largest reported change to date in the lysosomal pH range. As a point of comparison, the recently reported lifetime-based RpHLuorin2 sensor for acidic vesicles exhibits a total change in lifetime of 0.4 ns in the same pH range

[<a class="citation" href="http://doi.org/10.1021/acschembio.1c00907" title="Fluorescence Lifetime Imaging of pH along the Secretory Pathway; Peter T. A. Linders, Melina Ioannidis, Martin ter Beest, Geert van den Bogaart; ACS Chemical Biology 17, 1, 240-251 (2022)">Linders 2022</a>].

All other things being equal, mScarlet’s 4x increase in dynamic range relative to RpHluorin2 would enable detection of 4x smaller changes in pHlys, making mScarlet an excellent choice for a pHlys biosensor.</p>

<figure id="pH_sensitivity_main_figure">
<img src="./figures/fig1_Scarlet_pH_sensitivity/fig1_pH_sensitivity.png" alt="mScarlet pH Sensitivity" id="pH_sensitivity_main">
 <figcaption><strong>Figure 1: mScarlet lifetime reports pH in vitro and in lysosomes.</strong>

 <strong>(A)</strong> Lifetime changes in purified mScarlet over the pH range 4 to 7.5. Solid line indicates fit to a 4-parameter logistic model. 

 <strong>(B)</strong> Lifetime changes observed in U2OS cells transiently transfected with mScarlet-LAMP1. pHlys was equilibrated with the pH of the imaging buffer by incubation with 10 &mu;M nigericin and 2 &mu;M monensin in a high K<sup>+</sup> solution. pHlys was quantified as the average mean arrival time across all lysosomes in each cell. Error bars indicate mean ± S.D. over n=15 cells per pH. Solid line indicates fit to a 4-parameter logistic model.

 <strong>(C)</strong> Violin plots of the U2OS cell data from (B) by lysosome, where the median and first and third quartiles are shown with dotted lines across the plot. Adjacent or overlapping lysosomes were treated as one group, and lysosomal regions of interest smaller than 2 pixels were omitted. Number of identified lysosomes per condition (in ascending order by pH): 515, 976, 1532, 1426, 1287, 1589, 1998, 1746.

 <strong>(D)</strong> Lifetime (τ)-intensity overlay images of U2OS expressing mScarlet-LAMP1 with their pHlys set to the indicated value by ionophores. Intensity of each image is scaled to its own maximum and should not be compared between images.
 </figcaption>
</figure><p></p>

<p>On a technical note regarding mScarlet lifetime, we observe a noticeable change in not only the rate but also the shape of the mScarlet fluorescence decay (<a href="./appendix.html#raw_decay_pH_header">Fig. S1</a>) from pH 7.5 to 4. At high pH, mScarlet lifetime is approximately monoexponential, but at pH 4, we observe a multiexponential decay. This change in decay model poses a challenge for fitting, as FLIM data ideally should be fit to the same decay model in all conditions. To avoid artifacts from using different decay models, we opted to quantify all mScarlet lifetimes here with the mean photon arrival time (the first moment of the exponential distribution), which does not require a fit model.</p>

<p>Although fluorescence lifetime is generally more robust than fluorescence intensity, it still may respond to factors other than pH

[<a class="citation" href="http://doi.org/10.1021/cr900343z" title="Fluorescence Lifetime Measurements and Biological Imaging; Mikhail Y. Berezin, Samuel Achilefu*; Chemical Reviews 110, 5, 2641-2684 (2010)">Berezin 2010</a>].

To check for this, we characterized the response of mScarlet lifetime to a panel of other analytes in vitro. We do not observe a change in lifetime with mScarlet concentration (<a href="./appendix.html#mscarlet_conc_header">Fig. S2</a>), solution ionic strength, or concentration of Na<sup>+</sup>, Ca<sup>2+</sup>, Mg<sup>2+</sup>, K<sup>+</sup>, or Cl<sup>-</sup> (<a href="./appendix.html#mscarlet_ions_header">Fig. S3</a>). We also observe nearly identical pH dependencies in two different buffers (phosphate buffer with added NaCl and citrate-phosphate-borate buffer with added KCl, <a href="./appendix.html#mscarlet_temp_header">Fig. S4</a>). In our experience, temperature frequently affects fluorescence lifetime, but we found that changes in mScarlet lifetime with temperature were minimal in the range tested (23°C to 35°C). We observe a slight decrease in lifetime at high glycerol concentrations, which is likely attributable to an increase in refractive index (<a href="./appendix.html#mscarlet_glycerol_header">Fig. S5</a>, discussed further in <a href="./appendix.html#glycerol_text">Supplementary Text</a>). <strong>Overall, we find mScarlet lifetime to be unusually “well-behaved,” exhibiting minimal off-target sensitivity.</strong></p>

<h4>mScarlet lifetime reports lysosomal pH in mammalian cells</h4>

<p>Encouraged by the excellent pH sensitivity of mScarlet lifetime in vitro, we targeted mScarlet to the lysosomal lumen by fusing it to the N terminus of the lysosomal membrane protein LAMP1. We selected LAMP1 as an anchor because it has been characterized and used successfully in previous pHlys sensing constructs, where it was shown to minimally perturb lysosomal properties

[<a class="citation" href="http://doi.org/10.1091/mbc.e20-06-0383" title="pHLARE: A New Biosensor Reveals Decreased Lysosome pH in Cancer Cells; Bradley A. Webb, Francesca M. Aloisio, Rabab A. Charafeddine, Jessica Cook, Torsten Wittmann, Diane L. Barber; Molecular Biology of the Cell 32, 2, 131-142 (2021)">Webb 2021</a>, 

<a class="citation" href="http://doi.org/10.1080/15548627.2020.1771858" title="Live Imaging of Intra-Lysosome pH in Cell Lines and Primary Neuronal Culture Using a Novel Genetically Encoded Biosensor; Amy H. Ponsford, Thomas A. Ryan, Andrea Raimondi, Emanuele Cocucci, Susanne A. Wycislo, Florian Fröhlich, Laura E. Swan, Massimiliano Stagi; Autophagy 17, 6, 1500-1518 (2020)">Ponsford 2020</a>].

We also included a prolactin signal sequence for targeting of glycosylated membrane proteins

[<a class="citation" href="http://doi.org/10.1016/s0021-9258(17)34224-2" title="A Signal Sequence for the Insertion of a Transmembrane Glycoprotein. Similarities to the Signals of Secretory Proteins in Primary Structure and Function; Vishwanath R. Lingappa, Flora N. Katz, Harvey F. Lodish, Gunter Blobel; Journal of Biological Chemistry 253, 24, 8667-8670 (1978)">Lingappa 1978</a>]

and a linker sequence between mScarlet and LAMP1 from the previously published pHlys sensor pHLARE

[<a class="citation" href="http://doi.org/10.1091/mbc.e20-06-0383" title="pHLARE: A New Biosensor Reveals Decreased Lysosome pH in Cancer Cells; Bradley A. Webb, Francesca M. Aloisio, Rabab A. Charafeddine, Jessica Cook, Torsten Wittmann, Diane L. Barber; Molecular Biology of the Cell 32, 2, 131-142 (2021)">Webb 2021</a>].

</p>

<p>We then expressed this mScarlet-LAMP1 construct in U2OS cells via transient transfection and observed good lysosomal localization, as evidenced by colocalization with the SiR lysosome stain (<a href="./appendix.html#sir_lysosome_coloc_header">Fig. S6</a>). We also observe weak mScarlet-LAMP1 fluorescence at the plasma membrane, which has been attributed to lysosomal exocytosis

[<a class="citation" href="http://doi.org/10.1016/s0092-8674(01)00421-4" title="Plasma Membrane Repair Is Mediated by Ca2+-Regulated Exocytosis of Lysosomes; Anita Reddy, Elisabet V. Caler, Norma W. Andrews; Cell 106, 2, 157-169 (2001)">Reddy 2001</a>].

As is common with transient transfection, we observed high variance in expression level across the cell population, and cells with very high expression displayed irregular endolysosomal morphology. Nevertheless, approximately half of the cells displayed normal endolysosomal morphology and sufficiently bright fluorescence for lifetime imaging; we imaged these cells. In separate experiments, we also tested lentiviral expression methods, which produced more consistent, lower expression levels (see <a href="#A549_expression_text">below</a>).</p>

<p>To determine the pH sensitivity of the mScarlet-LAMP1 fusion, we used an ionophore cocktail to control pHlys while we measured the fluorescence lifetime (<a href="#pH_sensitivity_main_figure" target="_self">Figure 1B-D</a>). Lysosomes were permeabilized with 10 &mu;M nigericin and 2 &mu;M monensin for 10 minutes in high K<sup>+</sup> imaging solutions ranging from pH 4.0 to 7.5. Similar to the in vitro studies, we observed a large change in lifetime (1.5 ns from pH 4 to 7.5), with low between-cell variability at each pH. Interestingly, we observed somewhat different exact values for mScarlet lifetime in lysosomes versus in pure protein (see <a href="./appendix.html#in_vitro_vs_cells_text">Supplementary Text</a> for further discussion).</p>

<p>Fluorescence lifetime imaging often requires longer exposures than fluorescence intensity imaging, as many photons must be collected to accurately determine the lifetime. Prior to collecting these data, we were concerned that lifetime might not be a viable technique in lysosomes, which are small and rapidly moving. Fortunately, the high brightness and red-shifted spectrum of mScarlet enabled us to quantify lifetime at individual lysosomes (<a href="#pH_sensitivity_main_figure" target="_self">Fig. 1C, D</a>). The lifetime within and between ionophore-treated lysosomes was quite uniform, indicating low noise in the measurement. This is in contrast to our experience using two domain ratiometric biosensors, where we need to censor lysosomes displaying nonphysiological pH values, likely arising from preferential degradation of the normalizing or sensing domain (<a href="./appendix.html#two_color_misloc_header">Fig. S7</a>).</p>

<p>How small of a pHlys change can mScarlet-LAMP1 lifetime reliably quantify? This question is subtle because biosensors frequently exhibit nonlinear responses to analyte. Near the pKa, where mScarlet is most sensitive to pH, pH determinations will be more accurate because any noise will be smaller relative to the pH-related lifetime change. Conversely, at very high or very low pH, pHlys measurements will be less accurate. Importantly, the accuracy of the lifetime measurement will also depend on the number of photons collected. Therefore, improved pH resolution may be achievable at the cost of acquisition time. For these reasons, we have opted to report subsequent data in native lifetime units, along with any transformation into pH units.</p>

<p>Nevertheless, to get an estimate of resolution, we can approximate the lifetime-pH curve as linear near pH 5, with a slope of approximately 1 ns per pH unit in U2OS cells. We can then calculate the accuracy achieved with our acquisition settings from the standard deviation (SD) of the lifetime at a given ionophore-determined pH. For whole cell quantification, an SD of 0.1 ns is equivalent to 0.1 pH unit resolution near pH 5. For individual lysosome quantification, an SD of 0.2 ns is equivalent to 0.2 pH unit resolution near pH 5. These estimates are conservative, as we assumed the ionophores perfectly achieved the desired pH, but there likely is a small amount of variability even when the ionophores were present. </p>

<p id="A549_expression_text">To reduce variability in expression and investigate mScarlet-LAMP1 in another cell background, we generated a stable reporter line in A549 cells using lentiviral transduction. Versus transient transfection in U2OS, the polyclonal mScarlet-LAMP1 A549 line displayed lower average fluorescence and no cells with irregular lysosomal morphology. These differences are likely attributable to the lower gene copy number and different promoter used (SFFV in A549 vs. CMV in U2OS). Expectedly, this lower expression resulted in lower photon counts in our pHlys images. Even among the brightest 10% of cells, individual lysosome measurements had 50% more noise (SD of 0.3 ns at pH 5 among 1733 lysosomes) than in U2OS cells. We suspect that further fine-tuning of the expression level or isolation of a monoclonal line with desired brightness could improve this result. Nevertheless, we observed a similar relationship between mScarlet lifetime and pH in the ionophore calibration experiment in the A549 cells (<a href="./appendix.html#a549_calibration_header">Fig. S8</a>, <a href="./appendix.html#scarlet_fit_table">Table S1</a>), suggesting that the mScarlet-LAMP1 construct is a versatile and portable sensor for pHlys.</p>

<h4>pHlys distributions at rest and in response to cellular perturbations</h4>

<p>With this calibration in hand, we measured the distribution of pHlys present in unperturbed U2OS cells in complete growth media. We observe considerable heterogeneity in the pHlys of individual lysosomes, which is visible by eye in a fluorescence lifetime image (<a href="#perturbations_main_figure" target="_self">Fig. 2A</a>, <a href="./appendix.html#extra_bafa_endpt_header">Fig. S9</a>). Incubation with the lysosomal V-ATPase inhibitor bafilomycin A1 (BafA) neutralizes the pHlys, leading to a longer lifetime and higher pH (<a href="#perturbations_main_figure" target="_self">Fig. 2A-C</a>, <a href="./appendix.html#bafa_image_averages_header">Fig. S10</a>). We estimate that the average pHlys in control U2OS cells is 4.9, whereas in BafA treated cells it is ≥6.5.</p>

<figure id="perturbations_main_figure">
<img src="./figures/fig2_perturbations/fig2_perturbations.png" alt="mScarlet pH Sensitivity" id="perturbations_main">
 <figcaption><strong>Figure 2. mScarlet-LAMP1 tracks pHlys response to perturbations.</strong>

 Representative lifetime-intensity overlay images of individual U2OS cells expressing mScarlet-LAMP1 and treated with <strong>(A)</strong> DMSO control or <strong>(B)</strong>100 nM bafilomycin A1 (BafA) for 5 hours.

 <strong>(C)</strong>Histograms of mScarlet lifetime on a per-lysosome basis in cells treated with DMSO or 100 nM BafA (n=20 cells in each condition, acquired from 4 samples over 2 days). 

 <strong>(D)</strong>Differences in mScarlet-LAMP1 lifetime quantified after 5 minutes of incubation in the specified DMEM formulation. DMEM base contained 4.5 g/L glucose but is otherwise unsupplemented. All other conditions contained 4.5 g/L glucose, 10% FBS, 1 mM Na pyruvate, and 1% penicillin-streptomycin, along with a glutamine (Gln) source. Fresh Gln media was supplemented with 4 mM Gln on the day of the experiment; Orig. Gln media contained the 4 mM Gln included by the manufacturer. GMax media was fresh complete DMEM with 2 mM GlutaMAX; spent GMax media had the same composition as GMax media but was on the cells for 24 hours prior to the experiment. The Kruskal-Wallis test was used to evaluate differences between groups (H=61, p=1e-12), followed by Dunn’s post hoc test with Holm’s correction. For clarity, only significant differences versus a media change into fresh complete media with GlutaMAX (GMax, first column) are shown. Bars indicate median ± SD of n=15-20 cells per condition, acquired from 3-4 samples over 2 days.
 </figcaption>
</figure><p></p>

<p>These mScarlet-LAMP1 lifetime images map pHlys across the cell, allowing us to quantify the pHlys distribution (<a href="#perturbations_main_figure" target="_self">Fig. 2C</a>). As expected, the addition of BafA makes pHlys more uniform within the cell, decreasing the standard deviation of mScarlet lifetime from 0.41 ns to 0.20 ns. This narrowing of the lifetime distribution with BafA indicates that different abundance or activity of the V-ATPase produced the majority of the pHlys heterogeneity at rest. As a caveat to interpreting these results, we note that there is likely some bias in our identification of lysosomes by segmenting the mScarlet-LAMP1 signal. This will necessarily affect the pHlys distribution we report in (at least) two key ways: (1) LAMP1 has been reported to stain nondegradative endolysosomal structures, so some of these may be included in our pHlys distribution

[<a class="citation" href="http://doi.org/10.1083/jcb.201711083" title="Characterization of LAMP1-Labeled Nondegradative Lysosomal and Endocytic Compartments in Neurons; Xiu‑Tang Cheng, Yu‑Xiang Xie, Bing Zhou, Ning Huang, Tamar Farfel‑Becker, Zu‑Hang Sheng; Journal of Cell Biology 217, 9, 3127-3139 (2018)">Cheng 2018</a>].

(2) Smaller lysosomes, especially those that are more acidic, may not have sufficient signal to be identified as lysosomes by our segmentation algorithm. If this poses a particular concern, addition of other, pH-insensitive lysosomal stains such as SiR lysosome could be added to refine lysosomal identification. </p>

<p>We went on to evaluate whether there were spatial patterns in pHlys across the cell. Previous reports

[<a class="citation" href="http://doi.org/10.1083/jcb.201507112" title="The Position of Lysosomes within the Cell Determines Their Luminal pH; Danielle E. Johnson, Philip Ostrowski, Valentin Jaumouillé, Sergio Grinstein; Journal of Cell Biology 212, 6, 667-692 (2016)">Johnson 2016</a>, 

<a class="citation" href="http://doi.org/10.1091/mbc.e20-06-0383" title="pHLARE: A New Biosensor Reveals Decreased Lysosome pH in Cancer Cells; Bradley A. Webb, Francesca M. Aloisio, Rabab A. Charafeddine, Jessica Cook, Torsten Wittmann, Diane L. Barber; Molecular Biology of the Cell 32, 2, 131-142 (2021)">Webb 2021</a>]

suggested that lysosomes in the cellular periphery might be deacidified, although some other studies have not reproduced that result

[<a class="citation" href="http://doi.org/10.1080/15548627.2020.1771858" title="Live Imaging of Intra-Lysosome pH in Cell Lines and Primary Neuronal Culture Using a Novel Genetically Encoded Biosensor; Amy H. Ponsford, Thomas A. Ryan, Andrea Raimondi, Emanuele Cocucci, Susanne A. Wycislo, Florian Fröhlich, Laura E. Swan, Massimiliano Stagi; Autophagy 17, 6, 1500-1518 (2020)">Ponsford 2020</a>].

In our dataset, we do not observe clear differences in pHlys between the periphery and internal cellular regions (<a href="./appendix.html#regional_distributions_header">Fig. S11</a>). A possible explanation for these different results among studies (using different pH biosensors) is that the two-color pHlys sensing systems can produce erroneous pH values due to degradation or mistargeting (as we observed in <a href="./appendix.html#two_color_misloc_header">Fig. S7</a>). An alternative explanation is that pHlys patterning across the cell occurs in cell lines or culture conditions different from the ones tested here. As such, the spatial organization of pHlys is worth revisiting in other model systems using a quantitative, single color pHlys sensor such as mScarlet-LAMP1.</p>

<p>Given the role of the lysosome in nutrient sensing, we investigated how changes in media or nutrient levels affect pHlys. We initially struggled with variability and irreproducibility in this experiment, as different batches of growth media produced different pHlys. We became concerned that perhaps glutamine (Gln) breakdown or issues with the stable Gln analog GlutaMAX (Gmax) in culture media were confounding our results. To determine the source of these issues, we grew U2OS cells in Gmax-containing DMEM, exchanged the media for DMEM with different supplements, incubated for 5 minutes, and then recorded pHlys with mScarlet-LAMP1 lifetime (<a href="#perturbations_main_figure">Fig. 2D</a>, <a href="./appendix.html#media_images_header">Fig. S12</a>). We observed dramatically elevated pHlys in fresh media supplemented by the manufacturer with Gln (Orig. Gln) versus supplemented fresh on the day of the experiment with Gln (Fresh Gln, median pHlys of 5.5 versus 4.9). In addition, media changing itself decreased the median pHlys from 5.1 to 4.9 (Spent Gmax vs. Gmax), even if the cells had only been in their current media for 24 hours. The removal of all DMEM supplements other than glucose (DMEM base) did not have a statistically significant effect on pHlys in this short time window (5 minutes).</p>

<p>A likely explanation for the striking pHlys elevation in media supplemented by the manufacturer with Gln is that Gln breakdown leads to the formation of ammonia, which rapidly accumulates in lysosomes and increases their pH. Importantly, none of the components we used were expired per manufacturer labeling. Similar changes in pHlys from glutamine breakdown products have been observed previously

[<a class="citation" href="http://doi.org/10.1073/pnas.75.7.3327" title="Fluorescence Probe Measurement of the Intralysosomal pH in Living Cells and the Perturbation of pH by Various Agents; Shoji Ohkuma, Brian Poole; Proceedings of the National Academy of Sciences of the United States of America 75, 7, 3327-3331 (1978)">Ohkuma 1978</a>],

and a quantification of Gln in commercial media revealed considerable levels of ammonia

[<a class="citation" href="http://doi.org/10.1016/0022-1759(93)90331-z" title="The Concentrations of Glutamine and Ammonia in Commercially Available Cell Culture Media; S. Heeneman , N.E.P. Deutz, W.A. Buurman; Journal of Immunological Methods 166, 1, 85-91 (1993)">Heeneman 1993</a>].

As such, we strongly suggest that studies looking at pHlys use stable Gln analogs (e.g. Gmax). The effect of media change on pHlys is also interesting and could be attributed to either removal of waste products or addition of fresh nutrients. We hope to see future systematic studies of the effect of individual nutrients on pHlys over time to clarify this mechanism.</p>

<h4>Tracking pHlys changes over time at the lysosome level</h4>

<p>We next sought to determine whether mScarlet lifetime could be used to track pHlys dynamics in living cells. Although fluorescence lifetime imaging is increasingly employed as a live-cell imaging technique

[<a class="citation" href="http://doi.org/10.1016/j.cbpa.2015.05.024" title="Quantitative Two-Photon Imaging of Fluorescent Biosensors; Gary Yellen, Rebecca Mongeon; Current Opinion in Chemical Biology 27, 24-30 (2015)">Yellen 2015</a>],

it requires relatively large numbers of photons and is not trivial to deploy in longitudinal imaging. However, the high brightness and large signal change of mScarlet allows for pHlys measurements with relatively few photons, making it both gentle enough and fast enough for longitudinal recordings. We recorded pHlys every 5 minutes for one hour in cells treated with either DMSO control or 100 nM BafA (<a href="#time_series_main_figure" target="_self">Fig. 3</a>, <a href="./appendix.html#bafa_image_averages_header">Fig. S10</a>, <a href="./appendix.html#more_bafa_ts_figure">Fig. S13</a>). Lifetimes recorded in control cells were stable over the 1 hour time period, and morphological changes corresponding to cell stress were not observed. Furthermore, we observed the expected increase in pHlys in the BafA treated cells, with the pHlys at 1 hour approaching the value recorded after a 5 hour incubation (<a href="perturbations_main_figure">Fig. 2</a>).</p>

<figure id="time_series_main_figure">
<img src="./figures/fig3_baf_time_series/fig3_baf_time_series.png" alt="Response to BafA Over Time" id="time_series_main">
 <figcaption><strong>Figure 3. Individual lysosomes’ response to BafA over time is heterogeneous</strong>

  Individual frames of a time series during treatment with <strong>(A)</strong> 0.1% DMSO or <strong>(B)</strong> 100 nM BafA in individual U2OS cells expressing mScarlet-LAMP1. DMSO or BafA was added at time 0. 

  <strong>(C)</strong> Histograms quantifying pHlys of individual lysosomes in the cells depicted in (A) and (B). Adjacent or overlapping lysosomes were not separated by segmentation, and lysosome ROIs smaller than 2 pixels were omitted. Images are lifetime-intensity overlays.
 </figcaption>
</figure><p></p>

<p>Analysis of pHlys of the individual lysosomes over time revealed surprising heterogeneity. Instead of a gradual deacidification of all lysosomes, there appear to be lysosomes that are particularly vulnerable to BafA and deacidify almost completely within 5-10 minutes. Another lysosomal population remains acidic for almost the entire duration of the recording (<a href="#time_series_main_figure" target="_self">Fig. 3B</a> and <a href="./appendix.html#more_bafa_ts_figure">Fig. S13</a>). This range of susceptibilities likely results from differences in ionic composition and ion channel expression on the various lysosomes. Once BafA stops the V-ATPase, the rate of proton flow through other transporters or channels will determine how fast the lysosomes deacidify

[<a class="citation" href="http://doi.org/10.1146/annurev-physiol-021014-071649" title="Lysosomal Physiology; Haoxing Xu, Dejian Ren; Annual Review of Physiology 77, 1, 57-80 (2015)">Xu 2015</a>].

The heterogeneity observed implies that the composition of channels and transporters differs considerably among lysosomes, suggesting that lysosomes may be more functionally heterogeneous than previously thought. The method described herein with mScarlet-LAMP1 provides both a tool and a clear motivation for follow-up investigations of lysosomal ionic heterogeneity. </p>

<p>The data obtained here provides a snapshot of pHlys, resolved by individual lysosome, at a series of time points. We did not track individual lysosomes over the one hour time course, although it would be interesting to do so. Because lysosomes move rapidly around the cell in three dimensions, it was not feasible with our current lifetime system (point-scanning confocal with time-correlated single photon counting) to track individual lysosomes in 3D. We emphasize that there is no fundamental principle preventing faster acquisition of lifetime data. Therefore, combining mScarlet-LAMP1 lifetime with a faster lifetime imaging setup, such as electro-optic wide-field FLIM

[<a class="citation" href="http://doi.org/10.1021/acsnano.1c04470" title="Resonant Electro-Optic Imaging for Microscopy at Nanosecond Resolution; Adam J. Bowman, Mark A. Kasevich; ACS Nano 15, 10, 16043-16054 (2021)">Bowman 2021</a>]  

or a frequency-domain FLIM lightsheet microscope

[<a class="citation" href="http://doi.org/10.1364/ol.42.001269" title="Functional in Vivo Imaging Using Fluorescence Lifetime Light-Sheet Microscopy; Claire A. Mitchell, Simon P. Poland, James Seyforth, Jakub Nedbal, Thomas Gelot, Tahiyat Huq, Gerhard Holst, Robert D. Knight, Simon M. Ameer-Beg; Optics Letters 42, 7, 1269-1272 (2017)">Mitchell 2017</a>],

could enable fast volumetric lifetime imaging and lysosome tracking. We look forward to future instrumentation developments for fast FLIM to facilitate lifetime-based pHlys recordings.</p>

<h3>Conclusions</h3>

<p>Here, we have demonstrated that mScarlet lifetime is a sensitive and quantitative readout of pH, with an emphasis on its applications in the lysosome. As the most sensitive lifetime-based pHlys reporter to date, mScarlet lifetime elucideates subtle pH differences among lysosomes at rest and under drug stress. Nevertheless, the molecular basis for this pHlys heterogeneity, as well as its functional significance, remains unclear and a topic for future research. By dissecting pHlys at the individual lysosome level, mScarlet-LAMP1 lifetime opens up new avenues of inquiry into fundamental biology of the lysosome. More generally, the success of mScarlet lifetime in a difficult sample such as the mammalian lysosome makes us optimistic about the broad potential for FLIM in quantitative imaging. We hope to see more studies using biosensor lifetime as a strategy for precisely measuring cellular properties.</p> 

<h3>Materials and Methods</h3>

<h4>Chemical Reagents</h4>

<p>All salts for buffers were obtained from Sigma Aldrich or Thermo Fisher Scientific unless otherwise noted. Bafilomycin A1 (BafA) was obtained as a solid from Cayman Chemical or as a ready-made DMSO solution from Sigma-Aldrich. Nigericin sodium salt was obtained from Sigma-Aldrich. Monensin was obtained from Enzo. Phosphate-buffered saline (PBS) used throughout this work was obtained from Corning (#21-040-CV) and contained (at 1x, pH 7.4, in mM): 154 NaCl, 1 KH<sub>2</sub>PO<sub>4</sub>, 6 NaH<sub>2</sub>PO<sub>4</sub>. </p>

<h4 id=cell_culture_methods>Mammalian Cell Culture</h4>

<p>U2OS and A549 cells were cultured in DMEM (Gibco #31053028, 4.5 g/L glucose) supplemented with 10% fetal bovine serum (FBS, Gibco Performance Plus, #16000-044), 1% penicillin-streptomycin (pen-strep), 1% (2 mM) GlutaMAX, and 1 mM sodium pyruvate. Cultures were maintained in a 5% CO<sub>2</sub>, 37°C incubator and were passaged when they reached 80-95% confluency. Briefly, cells were dissociated with TrypLE express cell dissociation reagent (Thermo Fisher), quenched in complete culture media, spun down (5 minutes at 200xg) and resuspended in complete media for passaging or plating. Cultures were tested monthly for mycoplasma contamination. </p>

<p>Prior to seeding cells for imaging, 8 well chambered #1.5 coverglass (Cell-Vis C8-1.5H-N) was pre-coated with fibronectin solution (Sigma-Aldrich, F1141). Fibronectin solution was diluted 100x in PBS without Ca<sup>2+</sup> or Mg<sup>2+</sup> and was added to coat the bottom of each chamber well. Chambers were incubated with fibronectin at 37°C for 1-8 hours and then washed three times in PBS before seeding of cells. For imaging, cells were seeded onto 8 well chambered coverglass at a density of 25,000 cells/well. Cells were imaged 24-48 hours after seeding.</p>

<p>For media change experiments, additional media were used and added to the cells 5-10 minutes before imaging. Complete DMEM supplemented by the manufacturer with glutamine (DMEM, Orig.) was obtained from Corning (#10-013-CV) and supplemented with 10% FBS (Gibco, same as above) and 1% pen-strep. DMEM base is Gibco #31053028 without any additional supplements (DMEM, 4.5 g/L glucose, no phenol red). For freshly added glutamine (Gln, Fresh), DMEM base was supplemented with 10% FBS (Gibco, same as above), 4 mM glutamine (mixed up or thawed fresh that morning), 1% pen-strep, and 1 mM pyruvate. </p>

<h4>Mammalian Expression of mScarlet-LAMP1 Biosensor</h4>

<p>Vectors expressing mScarlet-LAMP1 (pTwist-mScarlet-LAMP1, pcDNA3-mScarlet-LAMP1, pCDH-mScarlet-LAMP1) were generated using standard molecular cloning techniques. Sequences are available in the <a href="./appendix.html#sequences">Appendix.</a></p>

<p>U2OS cells were transiently transfected with the pTwist-mScarlet-LAMP1 or the pcDNA3-mScarlet-LAMP1 vector using the TransIT-LT1 Transfection Reagent (Mirus Bio) according to manufacturer protocols. We used 2 &mu;L of the LT1 reagent per &mu;g of plasmid DNA. Plasmid DNA for transfection was obtained by miniprep (Thermo Scientific GeneJET) from E. coli DH5&alpha; cells. Cells were generally transfected the day after initial seeding onto glass (see <a href="#cell_culture_methods" target="_self">above</a>) and imaged the day after transfection. We also obtained good results by initially seeding cells into 6 well tissue culture plates (500,000 cells/well) on day 0, transfecting on day 1, passaging on day 2 into 8 well chambers (as above), and imaging on day 3.</p>

<p>A549 cells stably expressing mScarlet-LAMP1 were generated by lentiviral transduction. Lentivirus was packaged in Lenti-X HEK293T cells (ClonTech) with ViraPower packaging system (Invitrogen) and added to the A549 culture in the presence of 6 &mu;g/mL polybrene (Milipore). Following infection, transduced cells were selected with blasticidin (5 &mu;g/mL). The polyclonal population was enriched for the top 10% of expressing cells by fluorescence assisted cell sorting on a BigFoot Cell Sorter (Thermo Fisher Scientific) using the “enrich” mode. To prepare for sorting, cells were trypsinized as for <a href="#cell_culture_methods" target="_self">cell culture</a> and resuspended at a concentration of 1-10 million cells/mL in PBS containing 5% FBS. Cells were collected into complete culture media, spun down, and plated into fresh culture media. Stably expressing polyclonal A549 lines were then maintained as described above.</p>

<h4 id="mscarlet_purification">mScarlet Protein Purification</h4>

<p>We expressed 6xHis-mScarlet for purification using the pRSET vector in E. coli BL21(DE3) (NEB, #C2527H). BL21(DE3) transformed cells from an agar plate were inoculated into 200 mL of autoinduction expression media (ThermoFisher, #K6803) supplemented with carbenicillin and shake-cultured for 16 hours at 37°C. The cell pellet was harvested by centrifugation at 8,000 RCF for 10 minutes and resuspended into 25 mL of ice-cold PBS. The cells were then lysed on ice by sonication and centrifuged at 30,000 RCF for 45 minutes. The supernatant was collected and vacuum filtered through a 0.2 &mu;M PES membrane before adding imidazole to a final concentration of 25 mM.</p>

<p>The supernatant was loaded onto a 1 mL HisTrap HP column (Cytiva, #17524701) equilibrated in PBS at a flow of 0.5 mL/min using an ÄKTA Pure liquid chromatography system (Cytiva, #29018228). The column was washed with 10 column volumes of PBS supplemented with 25 mM imidazole before eluting with a 25-300mM imidazole gradient over 30 column volumes. The main peak monitored by UV Abs280 was collected and injected onto a Superdex200 size exclusion chromatography column (Cytiva, #28990944) equilibrated in PBS at a flow of 0.5 mL/min using an ÄKTA Pure liquid chromatography system. The main peak was collected and pooled for downstream analysis.</p>

<p>Protein purity was assessed with analytical size exclusion chromatography, mass spectrometry, and SDS-PAGE. In <a href="./appendix.html#purity_characterization">the SDS-PAGE gel</a>, we observed a primary band at 28 kDa as well as a secondary pair of bands at 19 kDa and 9 kDa. These fragments were also detected by intact mass spectrometry and such fragments are commonly seen with red fluorescent proteins, in which the polypeptide backbone is cleaved during the chromophore maturation process but the two fragments remain covalently linked by the chromophore

[<a class="citation" href="http://doi.org/10.1016/j.sbi.2012.09.002" title="Red Fluorescent Proteins: Chromophore Formation and Cellular Applications; Atsushi Miyawaki, Daria M. Shcherbakova, Vladislav V. Verkhusha; Current Opinion in Structural Biology 22, 5, 679-688 (2012)">Miyawaki 2012</a>]. 

Purified 6xHis-mScarlet was kept in PBS pH 7 at 2.7 mg/mL at 4°C for short-term storage and -80°C for long-term storage.</p>

<h4 id="ionophore_methods">Ionophore Calibration of Lysosomal pH</h4>

<p>Cells were taken to the microscope in media, and 10 minutes before data collection, media was exchanged for a high K<sup>+</sup> imaging buffer containing ionophores (10 &mu;M nigericin and 2 &mu;M monensin) at a given pH. High K<sup>+</sup> imaging buffer contained, in mM: 140 KCl, 1 CaCl<sub>2</sub>, 1 MgCl<sub>2</sub>, 5 glucose, as well as 10% Carmody’s buffer (phosphate-citrate-borate universal buffer

[<a class="citation" href="http://doi.org/10.1021/ed038p559" title=" An Easily Prepared Wide Range Buffer Series; Walter R. Carmody; Journal of Chemical Education 38, 11, 559-560 (1961)">Carmody 1961</a>]) 

at the indicated pH. Each Carmody’s buffer solution and each final imaging solution were pH corrected to within 0.1 pH unit with concentrated HCl or KOH at room temperature.</p>

<p>We observed that transfer of cells into another saline imaging solution (e.g. Live Cell Imaging Solution from Thermo Fisher, #A14291DJ) reduced the efficacy of ionophores in controlling pHlys. As such, we recommend transferring cells directly from growth media into the high K<sup>+</sup> ionophore solution on the microscope before imaging.</p>

<h4>Treatment with Bafilomycin to Perturb Lysosomal V-ATPase</h4>

<p>For endpoint determinations of lysosomal pH following bafilomycin treatment, cells were incubated with either DMSO vehicle control or 100 nM bafilomycin for 5-6 hours and then imaged. A full media change was performed when the drug was added, and cells were imaged in the drug-containing media.</p>

<p>For time series involving drug treatment, cells were media changed into fresh complete media 15 minutes before the start of the time series. Drug or vehicle control was added as a 2x solution in additional fresh complete media at the indicated time.</p>

<h4 id=flim_methods>Fluorescence Lifetime Imaging Microscopy (FLIM)</h4>

<p>FLIM data were collected on a Leica SP8 point-scanning confocal microscope equipped with an LSM FLIM/FCS upgrade for time-correlated single photon counting (TCSPC) from PicoQuant. A handshake between the LAS X software (Leica) and SymPhoTime software (PicoQuant) was used to collect FLIM images. Fluorescence excitation was provided with the built-in white light laser tuned to 561 nm. A repetition rate of 40 MHz was used; average excitation power at the sample ranged from 10-15 &mu;W. Emission was collected through a 100x/1.4 NA oil immersion objective (HC PL APO CS2, Leica) onto the built-in HyD detectors in photon counting mode in the range 590-700 nm. Photon arrival times were determined by a PicoHarp 300 TCSPC module. The large separation between excitation wavelength and the start of the emission collection window was required to remove stray excitation light, and the built-in notch filter was also included to exclude 561 nm light from the emission path. The confocal pinhole was set to 1 Airy unit (AU) for 580 nm emission light. Because of limitations in the software interface between the Leica and PicoQuant components, we were not able to perform line accumulation. To mimic this functionality, we recorded frames with oversampled pixels (1024-4096 pixels on an approximately 50 &mu;m side of an image) and subsequently binned to the desired final size. Final pixel sizes are approximately 200 nm x 200 nm square. We used larger than Nyquist-sampled pixels to obtain more photons per pixel and improve accuracy of lifetime measurements. No smoothing was applied to the images after binning was performed. Scanning was performed with the built-in Leica non-resonant scanner at a rate of 100-600 (total frame time of 6-20 seconds, depending on the specific experiment).</p>

<p>Live cells were incubated on the microscope at 35°C using an okolab live cell imaging enclosure chamber around the microscope. When cells were in bicarbonate-buffered media, an atmosphere of 5% CO<sub>2</sub> was maintained in a stagetop okolab chamber (H201-LG).</p>

<p>We did not threshold images for particular numbers of photons per lysosome, but we estimated that we had approximately 30 photons per binned pixel in the dimmest segmented lysosomes, corresponding to a minimum of 50-100 photons per lysosome where we attempted to quantify lifetime. Mean arrival time was determined on a per-pixel basis after binning, either using the built-in functionality in SymPhoTime or in python. The “time zero” was taken as the midpoint of the rise in fluorescence.</p>

<h4>Image Processing and Segmentation</h4>

<p>FLIM data were exported from SymPhoTime as TIFF images with two channels (sum photon count and mean arrival time). For time series datasets, we generated these TIFF images with mean arrival time calculations manually using a <a href="https://github.com/AndrewGYork/tools/blob/master/picoquant_tttr.py">custom parser for the SymPhoTime TTTR file format</a>. 

For quantification of pHlys, these two channel (photon count-lifetime) images were segmented into lysosomal pixels and background pixels (cellular and coverslip background) using a random forest classifier. Segmentation was performed on batches of 20-200 images at a time. <a href="https://github.com/jlazzaridean/useful_scripts/tree/main/mister_clicky_LIFETIME">Random forest classifier annotation code</a> is available on GitHub. All annotated images were manually checked by the user, and cell debris were removed where applicable.

Data were quantified with python scripts (available with our published dataset on <a href="https://doi.org/10.5281/zenodo.6115683">Zenodo</a>) as either the mean of all lysosomal pixels within the image (approximately one cell) or the mean of all lysosomal pixels within a segmented lysosomal region of interest. We did not attempt to separate overlapping or touching lysosomes in the initial masks. Histograms of pHlys by lysosome are displayed with bin widths of 0.1 ns.

Throughout the text, all “lifetime” images are displayed as a lifetime-intensity overlay, rather than the lifetime data alone. Code to generate these images from a TIFF stack containing a photon count image and a mean arrival time image is available with the raw image data on <a href="https://doi.org/10.5281/zenodo.6115683">Zenodo</a>.

<h4>mScarlet in vitro Characterization</h4>

<p>Purified 6xHis-mScarlet was diluted to 0.01 mg/mL in the indicated solution (see <a href="./appendix.html#supplementary_figures">Supplementary Figures S1-S5</a>) in a total volume of 50 &mu;L in a 384 well plate with a #1.5 glass bottom (CellVis P383-1.5H-N). Concentrations of PBS higher than 1x were obtained using 10x PBS (Corning 46-013-CM, which contains (at 1x): 137 mM NaCl, 2.7 mM KCl, 1.8 mM KH<sub>2</sub>PO<sub>4</sub>, 12 mM NaH<sub>2</sub>PO<sub>4</sub>).</p>

<p>Fluorescence lifetime data were collected using the same instrument described in <a href="#flim_methods">Fluorescence Lifetime Imaging</a>, with modifications to the acquisition settings as follows. A 10x/0.4 NA air objective (HC PL APO CS2, Leica) was used, and photons were collected from a large field of view (581 or 1162 nm on a side, 512x512 pixels per image). Lower power was required for the concentrated protein solutions versus expression in cells; average power generally ranged from 0.5-5 &mu;W. Excitation wavelength (561 nm) and repetition rate (40 MHz) were the same as in the live cell experiments. The confocal pinhole was set to 1 AU for 580 nm emission. Frame accumulation was used, with images collected over 45 seconds (30 frame sweeps, scan speed 400). Mean arrival time data shown reflect the average across the entire field of view. Note that care was taken to record from a focal plane relatively high up into the mScarlet solution rather than immediately adjacent to the coverslip, as erroneous lifetimes could be recorded at the coverslip-solution interface.</p>

<h4>Determination of mScarlet pKa</h4>

<p>pKa was determined by fitting to a 4 parameter logistic model (eqn 1) in Python using the scipy.optimize.curve_fit function. Standard deviations for each parameter were obtained from the covariance matrix by taking the square root of the diagonals. In eqn. 1 below, &tau; is the lifetime, max is the maximum lifetime at high pH, min is the minimum lifetime at low pH, b is the slope factor (similar to Hill coefficient), and pKa is the inflection point of the curve. </p>

<div>
$$\begin{equation}
&tau; = \text{max} + \frac{\text{min} - \text{max}}{
1 + \left(\frac{\text{pH}}{\text{pKa}}\right)^b}
\end{equation}$$
</div>

<h4>Resource Availability</h4>

All raw and processed data used in this work are available on <a href="https://doi.org/10.5281/zenodo.6115683">Zenodo</a>. Plasmid DNA sequences and construct amino acid sequences are included in the <a href="./appendix.html#sequences">Appendix</a>.

<h3>Acknowledgements</h3>

<p>We thank Andrew G. York for helpful feedback and for assistance with use of his TCSPC file parsing code. We thank Diane Barber, Christos Kougentakis, and Francesca Aloisio from the University of California, San Francisco for helpful discussions. This work was supported financially by Calico Life Sciences LLC. We would like to thank many scientists at Calico Life Sciences LLC for helpful discussions throughout this project, in particular Voytek Okreglak, Daniel Gottschling, Mary McMahon, Kayley Hake, and Alfred Millett-Sikking.</p>

<h3>Author Contribution Statement</h3>

<p>M.I. designed the project and performed experiments. J.R.L.D. designed the project, performed experiments, analyzed data, and wrote the manuscript. M.C.I., J.C.K.W., and J.Y. performed experiments.</p>

<h3>
Open-source software</h3>
<p>
As always, our work here depends critically on the open-source software community. Among our crucial dependencies (listed with version numbers used in this work):
</p>

<ul>
 <li>
<a href="https://python.org">Python</a> (version 3.9.9)
 </li>
 <li>
<a href="https://numpy.org">Numpy</a> (version 1.21.4)

[<a class="citation" href="http://doi.org/10.1038/s41586-020-2649-2" title="Array programming with NumPy; Charles R. Harris, K. Jarrod Millman, Stéfan J. van der Walt, Ralf Gommers, Pauli Virtanen, David Cournapeau, Eric Wieser, Julian Taylor, Sebastian Berg, Nathaniel J. Smith, Robert Kern, Matti Picus, Stephan Hoyer, Marten H. van Kerkwijk, Matthew Brett, Allan Haldane, Jaime Fernández del Río, Mark Wiebe, Pearu Peterson, Pierre Gérard-Marchant, Kevin Sheppard, Tyler Reddy, Warren Weckesser, Hameer Abbasi, Christoph Gohlke &amp; Travis E. Oliphant; Nature 585, 357–362 (2020)">Harris 2020</a>]
 </li>
 <li>
<a href="https://scipy.org">Scipy</a> (version 1.7.2)

[<a class="citation" href="http://doi.org/10.1038/s41592-019-0686-2" title="SciPy 1.0: Fundamental Algorithms for Scientific Computing in Python; Pauli Virtanen, Ralf Gommers, Travis E. Oliphant, Matt Haberland, Tyler Reddy, David Cournapeau, Evgeni Burovski, Pearu Peterson, Warren Weckesser, Jonathan Bright, Stéfan J. van der Walt, Matthew Brett, Joshua Wilson, K. Jarrod Millman, Nikolay Mayorov, Andrew R. J. Nelson, Eric Jones, Robert Kern, Eric Larson, CJ Carey, İlhan Polat, Yu Feng, Eric W. Moore, Jake VanderPlas, Denis Laxalde, Josef Perktold, Robert Cimrman, Ian Henriksen, E.A. Quintero, Charles R Harris, Anne M. Archibald, Antônio H. Ribeiro, Fabian Pedregosa, Paul van Mulbregt, and SciPy 1.0 Contributors; Nature Methods, 17(3) 261-272 (2020)">Virtanen 2020</a>]
 </li>
 <li>
<a href="https://pandas.pydata.org">Pandas</a> (version 1.3.4)

[<a class="citation" href="http://doi.org/10.5281/zenodo.3509134" title="pandas-dev/pandas: Pandas; The pandas development team; Zenodo (2020)">pandas-dev 2020</a>]
 </li>
 <li>
<a href="https://seaborn.pydata.org">seaborn</a> (version 0.11.2)

[<a class="citation" href="https://doi.org/10.21105/joss.03021" title="seaborn: statistical data visualization; Michael L. Waskom; Journal of Open Source Software 6, 60, 3021 (2021)">Waskom 2021</a>]
 </li>
 <li>
<a href="https://matplotlib.org">Matplotlib</a> (version 3.5.0)

[<a class="citation" href="http://doi.org/10.1109/MCSE.2007.55" title="Matplotlib: A 2D graphics environment; J. D. Hunter; Computing in Science &amp; Engineering, 9 (3) 90-95, (2007)">Hunter 2007</a>]
 </li>
 <li>
<a href="https://github.com/cgohlke/tifffile/">tifffile</a> (2021.11.2)
 </li>
 <li>
<a href="https://imagej.net/software/fiji/">Fiji</a>

[<a class="citation" href="https://doi.org/10.1038/nmeth.2019" title="Fiji: an open-source platform for biological-image analysis; Johannes Schindelin, Ignacio Arganda-Carreras, Erwin Frise, Verena Kaynig, Mark Longair, Tobias Pietzsch, Stephan Preibisch, Curtis Rueden, Stephan Saalfeld, Benjamin Schmid, Jean-Yves Tinevez, Daniel James White, Volker Hartenstein, Kevin Eliceiri, Pavel Tomancak, Albert Cardona; Nature Methods 9, 7, 676-682 (2012)">Schindelin 2012</a>]
</li>
 <li>
<a href="https://ffmpeg.org">FFmpeg</a>
 </li>
</ul>

<p>We sincerely thank the authors of these projects for enabling our work.</p>

<h3>Appendix</h3>

<p>Additional details and discussion can be found in the <a href="./appendix.html">Appendix</a>, which is also referenced via hyperlinks throughout this article.</p>

<noscript>
<h3>Your browser doesn't seem to support Javascript. This document uses Javascript to generate a reference list, which would normally be displayed here. Either activate Javascript, or use the "Download PDF" link above if you want to see the reference list.</h3></noscript>

</section>
<footer>
  <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="./images/cc_by_4p0.png"></a>

  <p><small>Hosted on <a href="https://pages.github.com">GitHub Pages</a></small></p>
</footer>
</div>
 <!--[if !IE]><script>fixScale(document);</script><![endif]-->
</body>
</html>
 
